### PR TITLE
Remove source-build 2.2 => master auto merge

### PR DESCRIPTION
**Pull request by dseefeld**
/cc @dagood @crummel

### CODE REVIEW

**Comment by dseefeld** [comment]
@dagood Can you merge this?  I don't have write access to this repo.

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1852,30 +1852,6 @@
         }
       }
     },
-    // Automate opening PRs to merge dotnet org repos' release/2.2 changes into master branches
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/buildtools/blob/release/2.2/**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: Let's leave this one alone, buildtools hasn't changed as much.

Comment (dseefeld) [reply Maestro/subscriptions.json]: Updated to only remove the source-build trigger.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -1856,7 +1856,6 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/buildtools/blob/release/2.2/**/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: I suppose you can remove the `,`, too. 😄 The service is lenient, though.

**Pull request merged**